crates/rolldown/tests/esbuild/dce/drop_label_tree_shaking_bug_issue3311/artifacts.snap

```diff
@@ -10,7 +9,7 @@ snapshot_kind: text
 
 //#region entry.js
 const myFunc = () => {
-	;
+	debugger;
```

Comment:
Are these dev or prod build?

Reply:
I think that's an error in the logic of whatever code is producing this output. `TakeIn` should only be used to temporarily replace a node with a dummy, and then the dummy itself is replaced with a "real" node.

With the old manually-written `TakeIn`, the dummy generated when replacing a `Statement` was an `EmptyStatement`. But in the codegen-ed `TakeIn`, it's now a `DebuggerStatement` (just because `DebuggerStatement` is an earlier variant of `Statement`, so `TakeIn` codegen uses it).

This change shouldn't matter, because the dummy node should never end up in the final AST anyway.